### PR TITLE
Initial Commit -> ZCL

### PR DESCRIPTION
**Pull request by ziyuanliu**
changing all VTC/LTC => ZCL using electrum-zcl-old commits as reference
ref #4

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,7 +6,7 @@ dist/
 *.egg/
 /electrum.py
 contrib/pyinstaller/
-Electrum_VTC.egg-info/
+Electrum_ZCL.egg-info/
 gui/qt/icons_rc.py
 gui/vtc/icons_rc.py
```
Comment (ch4ot1c) [inline .gitignore]: Should be /zcl/

File: gui/kivy/main_window.py [hunk 2]
```diff
@@ -154,7 +154,7 @@ def on_history(self, d):
         self._trigger_update_history()
 
     def _get_bu(self):
-        return self.electrum_config.get('base_unit', 'VTC')
```
Comment (ch4ot1c) [inline gui/kivy/main_window.py]: case changed :(

Comment (ziyuanliu) [reply gui/kivy/main_window.py]: thanks for the catch!

File: .gitignore [hunk 3]
```diff
@@ -8,8 +8,8 @@ dist/
 contrib/pyinstaller/
 Electrum_ZCL.egg-info/
 gui/qt/icons_rc.py
-gui/vtc/icons_rc.py
-gui/vtc/style_rc.py
+gui/zlc/icons_rc.py
```
Comment (Zethirdone) [inline .gitignore]: ZCL

File: .gitignore [hunk 4]
```diff
@@ -8,8 +8,8 @@ dist/
...
+gui/zlc/style_rc.py
```
Comment (Zethirdone) [inline .gitignore]: zcl

File: contrib/build-wine/deterministic.spec [hunk 5]
```diff
@@ -14,7 +14,7 @@ home = 'C:\\electrum-zcl\\'
 # We don't put these files in to actually include them in the script but to make the Analysis method scan them for imports
 a = Analysis([home+'electrum-zcl',
               home+'gui/qt/main_window.py',
-              home+'gui/vtc/main_window.py',
+              home+'gui/zlc/main_window.py',
```
Comment (Zethirdone) [inline contrib/build-wine/deterministic.spec]: zcl

File: contrib/build-wine/prepare-wine.sh [hunk 6]
```diff
@@ -98,7 +98,7 @@ wine mingw-get install mingw32-libz
 
 printf "[build]\ncompiler=mingw32\n" > $WINEPREFIX/drive_c/Python27/Lib/distutils/distutils.cfg
 
-$PYTHON -m pip install vtc_scrypt
+$PYTHON -m pip install zlc_scrypt
```
Comment (Zethirdone) [inline contrib/build-wine/prepare-wine.sh]: zcl

File: electrum-zcl [hunk 7]
```diff
@@ -371,7 +371,7 @@ if __name__ == '__main__':
     if cmdname == 'gui':
         fd, server = daemon.get_fd_or_server(config)
         if fd is not None:
-            plugins = init_plugins(config, config.get('gui', 'vtc'))
+            plugins = init_plugins(config, config.get('gui', 'zlc'))
```
Comment (Zethirdone) [inline electrum-zcl]: zcl

File: gui/zcl/__init__.py [hunk 8]
```diff
@@ -50,14 +50,14 @@
     import icons_rc
 except Exception:
     print "Error: Could not find icons file."
-    print "Please run 'pyrcc4 icons.qrc -o gui/vtc/icons_rc.py', and reinstall Electrum"
+    print "Please run 'pyrcc4 icons.qrc -o gui/zlc/icons_rc.py', and reinstall Electrum"
```
Comment (Zethirdone) [inline gui/zcl/__init__.py]: zcl

File: gui/zcl/__init__.py [hunk 9]
```diff
@@ -50,14 +50,14 @@
...
     sys.exit(1)
 
 try:
     import style_rc
 except Exception:
     print "Error: Could not find style file."
-    print "Please run 'pyrcc4 style.qrc -o gui/vtc/style_rc.py', and reinstall Electrum"
+    print "Please run 'pyrcc4 style.qrc -o gui/zlc/style_rc.py', and reinstall Electrum"
```
Comment (Zethirdone) [inline gui/zcl/__init__.py]: zcl

File: gui/zcl/__init__.py [hunk 10]
```diff
@@ -109,7 +109,7 @@ def __init__(self, config, daemon, plugins):
         run_hook('init_qt', self)
 
     def init_stylesheet(self):
-        f = QFile(":vtcstyle/style.qss")
+        f = QFile(":zlcstyle/style.qss")
```
Comment (Zethirdone) [inline gui/zcl/__init__.py]: zcl

File: gui/zcl/main_window.py [hunk 11]
```diff
@@ -62,7 +62,7 @@
 from qrtextedit import ShowQRTextEdit
 from transaction_dialog import show_transaction
 from fee_slider import FeeSlider
-from vtctabwidget import ZclTabWidget
+from zlctabwidget import ZclTabWidget
```
Comment (Zethirdone) [inline gui/zcl/main_window.py]: zcl

File: lib/daemon.py [hunk 12]
```diff
@@ -261,7 +261,7 @@ def stop(self):
         DaemonThread.stop(self)
 
     def init_gui(self, config, plugins):
-        gui_name = config.get('gui', 'vtc')
+        gui_name = config.get('gui', 'zlc')
```
Comment (Zethirdone) [inline lib/daemon.py]: zcl

File: plugins/audio_modem/__init__.py [hunk 13]
```diff
@@ -3,5 +3,5 @@
 fullname =  _('Audio MODEM')
 description = _('Provides support for air-gapped transaction signing.')
 requires = [('amodem', 'http://github.com/romanz/amodem/')]
-available_for = ['qt', 'vtc']
+available_for = ['qt', 'zlc']
```
Comment (Zethirdone) [inline plugins/audio_modem/__init__.py]: zcl

File: plugins/cosigner_pool/__init__.py [hunk 14]
```diff
@@ -6,4 +6,4 @@
     _("Transactions are encrypted and stored on a remote server.")
 ])
 #requires_wallet_type = ['2of2', '2of3']
-available_for = ['qt', 'vtc']
+available_for = ['qt', 'zlc']
```
Comment (Zethirdone) [inline plugins/cosigner_pool/__init__.py]: zcl

File: style.qrc [hunk 15]
```diff
@@ -3,7 +3,7 @@
         <file>fonts/SourceSansPro-Regular.otf</file>
         <file>fonts/SourceCodePro-Regular.otf</file>
     </qresource>
-    <qresource prefix="vtcstyle">
+    <qresource prefix="zlcstyle">
```
Comment (Zethirdone) [inline style.qrc]: zcl

File: setup.py [hunk 16]
```diff
@@ -52,7 +52,7 @@
         'electrum_zcl',
         'electrum_zcl_gui',
         'electrum_zcl_gui.qt',
-        'electrum_zcl_gui.vtc',
+        'electrum_zcl_gui.zlc',
```
Comment (Zethirdone) [inline setup.py]: zcl

File: plugins/virtualkeyboard/__init__.py [hunk 17]
```diff
@@ -2,4 +2,4 @@
 
 fullname = 'Virtual Keyboard'
 description = '%s\n%s' % (_("Add an optional virtual keyboard to the password dialog."), _("Warning: do not use this if it makes you pick a weaker password."))
-available_for = ['qt', 'vtc']
+available_for = ['qt', 'zlc']
```
Comment (Zethirdone) [inline plugins/virtualkeyboard/__init__.py]: zcl

File: plugins/trezor/__init__.py [hunk 18]
```diff
@@ -4,5 +4,5 @@
 description = _('Provides support for TREZOR hardware wallet')
 requires = [('trezorlib','github.com/trezor/python-trezor')]
 registers_keystore = ('hardware', 'trezor', _("TREZOR wallet"))
-available_for = ['qt', 'cmdline', 'vtc']
+available_for = ['qt', 'cmdline', 'zlc']
```
Comment (Zethirdone) [inline plugins/trezor/__init__.py]: zcl

Comment (Zethirdone) [inline plugins/trezor/__init__.py]: zcl

Comment (Zethirdone) [inline plugins/trezor/__init__.py]: zcl

File: plugins/ledger/__init__.py [hunk 19]
```diff
@@ -4,4 +4,4 @@
 description = 'Provides support for Ledger hardware wallet'
 requires = [('btchip', 'github.com/ledgerhq/btchip-python')]
 registers_keystore = ('hardware', 'ledger', _("Ledger wallet"))
-available_for = ['qt', 'cmdline', 'vtc']
+available_for = ['qt', 'cmdline', 'zlc']
```
Comment (Zethirdone) [inline plugins/ledger/__init__.py]: zcl

File: plugins/labels/__init__.py [hunk 20]
```diff
@@ -5,5 +5,5 @@
     _("Save your wallet labels on a remote server, and synchronize them across multiple devices where you use Electrum."),
     _("Labels, transactions IDs and addresses are encrypted before they are sent to the remote server.")
 ])
-available_for = ['qt', 'kivy', 'vtc']
+available_for = ['qt', 'kivy', 'zlc']
```
Comment (Zethirdone) [inline plugins/labels/__init__.py]: zcl

File: plugins/keepkey/__init__.py [hunk 21]
```diff
@@ -4,4 +4,4 @@
 description = _('Provides support for KeepKey hardware wallet')
 requires = [('keepkeylib','github.com/keepkey/python-keepkey')]
 registers_keystore = ('hardware', 'keepkey', _("KeepKey wallet"))
-available_for = ['qt', 'cmdline', 'vtc']
+available_for = ['qt', 'cmdline', 'zlc']
```
Comment (Zethirdone) [inline plugins/keepkey/__init__.py]: zcl

File: plugins/email_requests/__init__.py [hunk 22]
```diff
@@ -2,4 +2,4 @@
 
 fullname = _('Email')
 description = _("Send and receive payment request with an email account")
-available_for = ['qt', 'vtc']
+available_for = ['qt', 'zlc']
```
Comment (Zethirdone) [inline plugins/email_requests/__init__.py]: zcl

File: plugins/digitalbitbox/__init__.py [hunk 23]
```diff
@@ -3,4 +3,4 @@
 fullname = 'Digital Bitbox'
 description = _('Provides support for Digital Bitbox hardware wallet')
 registers_keystore = ('hardware', 'digitalbitbox', _("Digital Bitbox wallet"))
-available_for = ['qt', 'cmdline', 'vtc']
+available_for = ['qt', 'cmdline', 'zlc']
```
Comment (Zethirdone) [inline plugins/digitalbitbox/__init__.py]: zcl

**Comment by ziyuanliu** [comment]
@Zethirdone @ch4ot1c thanks for the catch!

**Pull request merged**